opta/core/aws.py

```diff
@@ -167,6 +161,35 @@ def force_delete_terraform_lock_id(self) -> None:
             Key={"LockID": {"S": f"{bucket}/{self.layer.name}"}},
         )
 
+    @classmethod
+    def get_all_remote_configs(cls) -> Dict[str, Dict[str, "StructuredConfig"]]:
+        prefix = "opta_config/"
+        s3 = boto3.client("s3")
+        opta_config_detailed_map = {}
```

Comment:
let's use consistent naming all over, this method uses `StructuredConfig` and `remote_configs`
`detailed_map` `detailed_config` seem like similar concepts but using different names make the code harder to read.